### PR TITLE
Fix `parseParams`

### PR DESCRIPTION
**Pull request by valeriykundas**
Was getting `TypeError: Cannot read property 'length' of null` exception. Added `null` handler in addition.

*What is strange is these errors were not shown in devtools.*

### CODE REVIEW

**Review by Dizzzmas — APPROVED** [review]

**Review by revmischa — APPROVED** [review]

**Review by AnneMedvedeva — APPROVED** [review]

File: src/apiClient/index.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const parseParams = (params: object) => {
 
   keys.forEach(key => {
     const isParamTypeObject = typeof params[key] === 'object'
-    const isParamTypeArray = isParamTypeObject && params[key].length >= 0
+    const isParamTypeArray = isParamTypeObject && params[key] && params[key].length >= 0
```
Comment (d-spiridonov) [inline src/apiClient/index.ts:19]:
```suggestion
    const isParamTypeArray = isParamTypeObject && params[key]
```

Comment (d-spiridonov) [reply src/apiClient/index.ts:19]: It should be enough to do a check like this.

Comment (rkyslyy) [reply src/apiClient/index.ts:19]: `Array.isArray(params[key])`?

**Review by rkyslyy — APPROVED** [review]

**Review by ProbablyNextTime — APPROVED** [review]

**Review by adamszeptycki — CHANGES_REQUESTED** [review]
Can i ask for simple test for it?

**Pull request merged**